packages/shared/src/hooks/feed/useTrackImpression.ts

```diff
@@ -35,6 +35,9 @@ export default function useTrackImpression(
             columns,
             column,
             row,
+            extra: {
+              feedback: true,
+            },
```

Comment:
Just wanna add its crucial we test this is sent alright since its gonna be a way for platform team to do their logic.